logger/logger.go

```diff
@@ -0,0 +1,79 @@
+package logger
+
+import (
+	"fmt"
+	"log"
+)
+
+type Logger interface {
+	Debug(msg string)
+	Debugf(format string, args ...interface{})
+	Info(msg string)
+	Infof(format string, args ...interface{})
+	Warn(msg string)
+	Warnf(format string, args ...interface{})
+	Error(msg string)
+	Errorf(format string, args ...interface{})
+}
+
+func init() {
+	internalLogger = &LoggerImpl{}
```

Comment:
previous default is info level, this is error level.

Reply:
yeah that makes sense